src/types/Calculator.ts

```diff
@@ -42,6 +42,10 @@ export interface CalculationOptions {
   verboseCalculationResults?: boolean;
   /** if true trust the content of meta.profile as a source of truth for what profiles the data that cql-exec-fhir grabs validates against */
   trustMetaProfile?: boolean;
+  /** if true, cache ELM JSONs and associated data for access in subsequent runs within 10 minutes */
+  useElmJsonsCaching?: boolean;
+  /** if true, clears ElmJsons cache before running calculation */
+  clearElmJsonsCache?: boolean;
```

Comment:
Can you update the README adding information about these new calculation options?

Reply:
ooh good catch!